### ISSUE TITLE
another abandoned react native library?

### ISSUE THREAD

**Issue by SilencerWeb** [vault-development/react-native-svg-uri#139]

**Comment by SM-Wistful** [comment]
Yeah, I don't know if this one still works. I created a tool you can use to do this though: https://github.com/SM-Wistful/react-native-svg 

It's not particularly elegant but it works for me most of the time.
